### PR TITLE
[PROPIM-558] NEW HighlightSection

### PR DESCRIPTION
**Pull request by gabrielagabriel**
## Description

screencapture-localhost-4000-2021-01-22-12_55_21

## What has been done

- Add attribute/props `hidden` instead of `class` to hidden an `Item`
- Use `styled-components` syntaxe

## Things to consider

This is a DRAFT
Missing:
- RTL

### CODE REVIEW

File: src/layout/section/highlightSection/HighlightSection.story.mdx [hunk 1]
```diff
@@ -1,37 +1,195 @@
...
-  Section content
+export const content = [
+  {heading: "Top trajets en bus", items: initialItems},
+  {heading: "Top villes en bus", items: items}
+]
+
+<HighlightSection
+  content={content}
+  toggleLabel={{
+    on: "Show more",
```
Comment (Tuxosaurus) [inline src/layout/section/highlightSection/HighlightSection.story.mdx:189]: It's the opposite (on: Less, off: More)

File: src/layout/section/highlightSection/HighlightSection.story.mdx [hunk 2]
```diff
@@ -1,37 +1,195 @@
...
+  },
+  {
+    id: '3',
+    label: <RideAxis from="Bus Paris" to="Lille" />,
+    data: '18,99 €',
+    href: <a href="#" />,
+    ariaLabel: 'Aria label',
+  },
+  {
+    id: '1',
```
Comment (Tuxosaurus) [inline src/layout/section/highlightSection/HighlightSection.story.mdx:30]: update `id` (there's 1, 2 ,3, 1, 2, 3)

File: src/layout/section/highlightSection/HighlightSection.style.tsx [hunk 3]
```diff
@@ -1,13 +1,102 @@
...
+  grid-template-columns: 1fr;
+  grid-gap: ${space.m};
+  grid-auto-rows: minmax(100px, auto);
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+`
+export const Col = styled.li`
+  display: ${props => props.hidden && 'none'};
```
Comment (gabrielagabriel) [inline src/layout/section/highlightSection/HighlightSection.style.tsx:93]: `item.hidden` receive `display: none`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Unreleased
 
-- [...]
+- **[NEW]** Brand new `HighlightSection` component.
```
Comment (Tuxosaurus) [inline CHANGELOG.md:3]: Wondering if it should be BREAKING CHANGE (the HighlightSection component existed but it's all changed)

**Review by Tuxosaurus — APPROVED** [review]

**Pull request merged**